common/solana_rpc/alt_builder.py

```diff
@@ -150,6 +150,28 @@ def build_alt_tx_set(self, alt: SolAltInfo) -> SolAltTxSet:
 
         return SolAltTxSet(create_alt_tx_list=create_alt_tx_list, extend_alt_tx_list=extend_alt_tx_list)
 
+    def build_alt_custom_tx_set(self, alt: SolAltInfo) -> SolAltTxSet:
```

Comment:
`def build_alt_tx_set()` isn't used anymore
You should comment or remove it, just because it will confuse the future developer - why does it exist but not get used?